lib/flutter_map.dart

```diff
@@ -30,12 +29,12 @@ export 'package:flutter_map/src/layer/marker_layer.dart';
 export 'package:flutter_map/src/layer/overlay_image_layer.dart';
 export 'package:flutter_map/src/layer/polygon_layer.dart';
 export 'package:flutter_map/src/layer/polyline_layer.dart';
-export 'package:flutter_map/src/layer/tile_layer/coords.dart';
-export 'package:flutter_map/src/layer/tile_layer/tile.dart';
 export 'package:flutter_map/src/layer/tile_layer/tile_builder.dart';
+export 'package:flutter_map/src/layer/tile_layer/tile_coordinate.dart';
```

Comment:
Tile was renamed to TileImage.

This is the only addition, the rest are just re-ordered because my IDE does it automatically and the order had been broken. Given that only a small amount of exports were out of order I think it makes sense to re-order them.